### DISCUSSION TITLE
Dmenu mode does not appear, String literal not valid UTF-8 string Please help.....

### DISCUSSION THREAD

**Discussion by DarkIce000** [davatorium/rofi#1907] [Q&A]
darkice@darkice-Inspiron-3542:~$ rofi -dmenu
xkbcommon: ERROR: /usr/share/X11/locale/iso8859-1/Compose:39:34: string literal is not a valid UTF-8 string
xkbcommon: ERROR: /usr/share/X11/locale/iso8859-1/Compose:40:29: string literal is not a valid UTF-8 string
xkbcommon: ERROR: /usr/share/X11/locale/iso8859-1/Compose:41:29: string literal is not a valid UTF-8 string
xkbcommon: ERROR: /usr/share/X11/locale/iso8859-1/Compose:42:29: string literal is not a valid UTF-8 string
xkbcommon: ERROR: /usr/share/X11/locale/iso8859-1/Compose:43:29: string literal is not a valid UTF-8 string
xkbcommon: ERROR: /usr/share/X11/locale/iso8859-1/Compose:44:27: string literal is not a valid UTF-8 string
xkbcommon: ERROR: /usr/share/X11/locale/iso8859-1/Compose:45:27: string literal is not a valid UTF-8 string
xkbcommon: ERROR: /usr/share/X11/locale/iso8859-1/Compose:46:27: string literal is not a valid UTF-8 string
xkbcommon: ERROR: /usr/share/X11/locale/iso8859-1/Compose:47:27: string literal is not a valid UTF-8 string
xkbcommon: ERROR: /usr/share/X11/locale/iso8859-1/Compose:48:29: string literal is not a valid UTF-8 string
xkbcommon: ERROR: /usr/share/X11/locale/iso8859-1/Compose:49:29: string literal is not a valid UTF-8 string
xkbcommon: ERROR: /usr/share/X11/locale/iso8859-1/Compose:49:29: too many errors
xkbcommon: ERROR: /usr/share/X11/locale/iso8859-1/Compose:49:29: failed to parse file

these are the errors i am getting when i  try to run the rofi in dmenu mode. in the terminal.
i have installed rofi and i3 wm in linux mint 22

 ----------------------------- 
      .-MMMM`..-:::::::-..`MMMM-.          OS: Linux Mint 21.2 x86_64 
    .:MMMM.:MMMMMMMMMMMMMMM:.MMMM:.        Host: Inspiron 3542 
   -MMM-M---MMMMMMMMMMMMMMMMMMM.MMM-       Kernel: 5.15.0-86-generic 
 `:MMM:MM`  :MMMM:....::-...-MMMM:MMM:`    Uptime: 17 mins 
 :MMM:MMM`  :MM:`  ``    ``  `:MMM:MMM:    Packages: 2502 (dpkg) 
.MMM.MMMM`  :MM.  -MM.  .MM-  `MMMM.MMM.   Shell: bash 5.1.16 
:MMM:MMMM`  :MM.  -MM-  .MM:  `MMMM-MMM:   Resolution: 1366x768 
:MMM:MMMM`  :MM.  -MM-  .MM:  `MMMM:MMM:   WM: i3 
:MMM:MMMM`  :MM.  -MM-  .MM:  `MMMM-MMM:   Theme: Mint-Y-Dark [GTK2/3] 
.MMM.MMMM`  :MM:--:MM:--:MM:  `MMMM.MMM.   Icons: Mint-Y-Sand [GTK2/3] 
 :MMM:MMM-  `-MMMMMMMMMMMM-`  -MMM-MMM:    Terminal: gnome-terminal 
  :MMM:MMM:`                `:MMM:MMM:     CPU: Intel i3-4005U (4) @ 1.700GHz 
   .MMM.MMMM:--------------:MMMM.MMM.      GPU: Intel Haswell-ULT 
     '-MMMM.-MMMMMMMMMMMMMMM-.MMMM-'       Memory: 1991MiB / 3816MiB

**Comment by DaveDavenport** [comment]
This is an external library that throws an error.  See https://github.com/davatorium/rofi/issues/814
